src/main/java/com/spotify/docker/client/messages/ContainerMount.java

```diff
@@ -60,29 +61,15 @@ public boolean equals(final Object o) {
 
     final ContainerMount that = (ContainerMount) o;
 
-    if (source != null ? !source.equals(that.source) : that.source != null) {
-      return false;
-    }
-    if (destination != null ? !destination.equals(that.destination) : that.destination != null) {
-      return false;
-    }
-    if (mode != null ? !mode.equals(that.mode) : that.mode != null) {
-      return false;
-    }
-    if (rwInfo != null ? !rwInfo.equals(that.rwInfo) : that.rwInfo != null) {
-      return false;
-    }
-
-    return true;
+    return Objects.equal(this.source, that.source) &&
+        Objects.equal(this.destination, that.destination) &&
+        Objects.equal(this.mode, that.mode) &&
+        Objects.equal(this.rw, that.rw);
   }
 
   @Override
   public int hashCode() {
-    int result = source != null ? source.hashCode() : 0;
-    result = 31 * result + (destination != null ? destination.hashCode() : 0);
-    result = 31 * result + (mode != null ? mode.hashCode() : 0);
-    result = 31 * result + (rwInfo != null ? rwInfo.hashCode() : 0);
-    return result;
+    return Objects.hashCode(source, destination, mode, rw);
```

Comment:
this can just be `java.util.Objects.hash(..)` (we should replace the usage in the rest of the codebase later too)